server/form-pages/apply/risk-and-need-factors/access-and-healthcare/accessNeedsFurtherQuestions.ts

```diff
@@ -10,8 +10,8 @@ import AccessNeeds from './accessNeeds'
 
 export type AccessNeedsFurtherQuestionsBody = {
   needsWheelchair: YesOrNo
-  mobilityNeeds: string
-  visualImpairment: string
+  healthConditions: YesOrNo
```

Comment:
could we use the YesOrNoWithDetail utility type here?

Reply:
Oh course! Nice catch! 👍